source/api_documentation/transactions_api/_network_optional_param_table.rst

```diff
@@ -0,0 +1,411 @@
+
+  * - dynamic_number_pool_referrer_param5_name
+    - Pool Param 5 Name
+    - [Number pooling links only] The name for parameter 5
+
+  * - dynamic_number_pool_referrer_param5_value
+    - Pool Param 5 Value
+    - [Number pooling links only] The value for parameter 5
+
+  * - dynamic_number_pool_referrer_param6
+    - Dynamic Number Pool Referrer Param6 Value
+    - [Number pooling links only] name=value for parameter 6, url escaped. Example: name%3Dvalue
+
+  * - dynamic_number_pool_referrer_param6_name
+    - Pool Param 6 Name
+    - [Number pooling links only] The name for parameter 6
+
+  * - dynamic_number_pool_referrer_param6_value
+    - Pool Param 6 Value
+    - [Number pooling links only] The value for parameter 6
+
+  * - dynamic_number_pool_referrer_param7
+    - Dynamic Number Pool Referrer Param7 Value
+    - [Number pooling links only] name=value for parameter 7, url escaped. Example: name%3Dvalue
+
+  * - dynamic_number_pool_referrer_param7_name
+    - Pool Param 7 Name
+    - [Number pooling links only] The name for parameter 7
+
+  * - dynamic_number_pool_referrer_param7_value
+    - Pool Param 7 Value
+    - [Number pooling links only] The value for parameter 7
+
+  * - dynamic_number_pool_referrer_param8
+    - Dynamic Number Pool Referrer Param8 Value
+    - [Number pooling links only] name=value for parameter 8, url escaped. Example: name%3Dvalue
+
+  * - dynamic_number_pool_referrer_param8_name
+    - Pool Param 8 Name
+    - [Number pooling links only] The name for parameter 8
+
+  * - dynamic_number_pool_referrer_param8_value
+    - Pool Param 8 Value
+    - [Number pooling links only] The value for parameter 8
+
+  * - dynamic_number_pool_referrer_param9
+    - Dynamic Number Pool Referrer Param9 Value
+    - [Number pooling links only] name=value for parameter 9, url escaped. Example: name%3Dvalue
+
+  * - dynamic_number_pool_referrer_param9_name
+    - Pool Param 9 Name
+    - [Number pooling links only] The name for parameter 9
+
+  * - dynamic_number_pool_referrer_param9_value
+    - Pool Param 9 Value
+    - [Number pooling links only] The value for parameter 9
+
+  * - dynamic_number_pool_referrer_referrer_campaign
+    - AdWords Campaign
+    - AdWords Campaign name, provided by Google
+
+  * - dynamic_number_pool_referrer_referrer_campaign_id
+    - AdWords Campaign ID
+    - AdWords Campaign ID, provided by Google
+
+  * - dynamic_number_pool_referrer_search_engine
+    - Traffic Source
+    - [Number pooling links only] Search engine used.
+
+  * - dynamic_number_pool_referrer_search_keywords
+    - Keywords
+    - [Number pooling links only] Search keywords used
+
+  * - dynamic_number_pool_referrer_search_keywords_id
+    - AdWords Keywords ID
+    - AdWords Keyword ID, provided by Google
+
+  * - dynamic_number_pool_referrer_search_type
+    - Search Type
+    - [Number pooling links only] "Paid" or "Organic".
```

Comment:
can you do `include`'s from within these rst files? If so I would consider making the 30+ dynamic_nuber_pool* fields into its own rst file that each of the three could pull in. Same with the CPA columns and others that is a group of 10+ that show up in all reports. That way the lists of optional params per org type would hopefully read pretty short in the source